kolibri/plugins/management/assets/src/views/user-page/user-create-modal.vue

```diff
@@ -54,7 +54,7 @@
 
       <!-- Button Options at footer of modal -->
       <section class="footer">
-        <k-button :text="$tr('createAccount')" :primary="true" :loading="loading"/>
+        <k-button :text="$tr('createAccount')" :primary="true" :loading="loading" type="submit"/>
```

Comment:
How does this work? I don't see`type` as props on `k-button`, and I'm having trouble finding documentation on how form `submit` events work.

(Incidentally, don't see `loading` either.)

Handling form submission has been a tricky, recurring problem to avoid double-submits and other edge cases. I'm not sure what the right solution is, but now might be a good opportunity to document best practice in our Form and Button style guide pages.